Testscripts/Linux/validate-intel-sgx-driver.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-intel-sgx-driver.sh
+# Description:
+#    Check the existence of Intel sgx driver.
+#    Install Open-Enclave and build&run Samples
+# Supported Distros:
+#    Ubuntu
+#######################################################################
+
+set -e
+set -x
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" >state.txt
+    exit 1
+}
+
+UtilsInit
+
+sudo apt-get update
+sudo apt-get -y upgrade
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
```

Comment:
does this repo work against all ubuntu image, like 16.04, 19.10?

Reply:
@Francis-Liu By referring other test script, there is if-statement condition to filter out for specific distro or version. I suggest you borrow that method, if this is the only version we test.

Reply:
This script is meant for 'Canonical UbuntuServer 18.04-LTS Latest' only. In my testing, -ARMImageName 'Canonical UbuntuServer 18.04-LTS Latest' is specified. How do I specify this in test definition xml?

Reply:
As far as I understand, you will do PS1 wrapper based on @harz566 suggestion. This is example for you, https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1#L77

Reply:
Besides UBUNTU, how to further filter version != 18.04?

Reply:
Acked. Make sense,